### ISSUE TITLE
[BUG] how to update a field value of a ressource submission

### ISSUE THREAD

**Issue by gismomtl** [formio/formio#1019]
## Environment
Please provide as many details as you can:

- Hosting type
  - [x] Form.io

Hi, 
I'm trying to update an existing ressource submission via a form.
a "ticket" ressource contain unique id (ticketid), a status and other fields (ex. name of the customer,..)
I've a "update status form" which retrieve the list of ticket, get the id of the ticket and have a field "status update".
In the action, I set the save action (save to ressource) and specify the ticket id and the status as link to the form fields with "update" method.

However I'm not able to have the ticket in the ressource to be update (just the status, without changing the other fields)
I tried to change access rights on the ressource but it's not changing anything

ticket resource: https://vwclcxrndomvfvv.form.io
form: https://formview.io/#/vwclcxrndomvfvv/testcheck&header=1

**Comment by gismomtl** [comment]
anyone has a solution??

**Comment by gismomtl** [comment]
any news?

**Comment by randallknutson** [comment]
Have you tried using a nested form component?

**Comment by gismomtl** [comment]
I tried but no luck.. do you have any working example? 
such requirement should it be possible?

**Comment by randallknutson** [comment]
Forms generally manage their own data, not data for another resource. The best thing to look at is either nested forms or the select component with the ability to add another resource. You can also check out the Save Submission action where you can save to a different resource.

**Comment by gismomtl** [comment]
Yes that's exactly what I trying to do: use a form with "Save Action" to save to the selected ressource and update only a few fields of a ressource submission.. but I'm not able to do so... whatever option selected in the "save to ressource" action (after/before, update/create,...)

**Comment by gismomtl** [comment]
any way to confirm that "save to ressource" should be able to update existing ressource submission? any working example available??

**Comment by randallknutson** [comment]
Save to Resource should be able to update existing submissions. This will only work if you also have two "Save Submission" actions. One to save to the primary form and the second to the external resource. Then what happens is the submission id in the external resource is save to the submission in the primary form. Any subsequent updates to the form submission has the id of the linked submission in the resource and can do the update accordingly.

**Comment by gismomtl** [comment]
I tried online https://pro.formview.io/#/ptmghxvybnwhhhp/ressource1?header=1&reset=1
I can save new TicketID in the ressource.
but on the ressource:
- if it set TicketID not unique, everytime I set a new status to an existing TicketID value, a new submission in the ressource is created .. but I want to update the existing submission, not create a new one.
- if it set TicketID as unique, a form submission with an existing ticketID is refused ""ticketid" must be unique."

**Comment by randallknutson** [comment]
I took a closer look and realized there is a "Resource Property" option when saving to a different resource. This is to store the resource id so that the update can be made correctly.

**Comment by gismomtl** [comment]
the "Resource Property" allow to save the resource item in the submission under the field name selected as "resource property". However it doesnt help to allow update of the existing resource item... any other solution? are you able to update a resource submission from a form ?

**Comment by wag110894** [comment]
@gismomtl, we will need to investigate this on our side on how to accomplish this for you. We will let you know what we find.

**Comment by wag110894** [comment]
@gismomtl,

Thank you for your patience. We will need to investigate this issue as part of a support package. Please contact support@form.io for more information.

Thank you.

**Comment by gismomtl** [comment]
do you have any news on your side ? have you been able to reproduce the issue & any target version in which it will be solved?

**Comment by wag110894** [comment]
Hello @gismomtl,
As I mentioned back on July 13th, we would need to investigate this issue as part of a support package.

**Comment by centfis** [comment]
Hi I'm new to formio forms. I need to create Drop-down box dependent on the option selected in another drop-down box. I started following [https://help.form.io/tutorials/walkthroughs/dynamic-select-filtering/](url) this link to create what I'm looking for. 
 I added a 'select' field in my newly created form and chose the 'Data Source Type' as 'Resource'. Now I'm not able to enter any field value or resource value. Have anyone experienced this before? Can you guide me how to do it?
